### PR TITLE
Add cheqd method

### PR DESCRIPTION
**Pull request by Toktar**
Work in progress

### CODE REVIEW

File: packages/did-core-test-server/suites/implementations/dereferencer-cheqd.json [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
...      "contentStream": "{\"@context\": \"https://www.w3.org/ns/did/v1\",\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY\",\"authentication\":[\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#key1\"],\"service\":[{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#website\",\"serviceEndpoint\":\"https://www.cheqd.io\",\"type\":\"LinkedDomains\"},{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#non-fungible-image\",\"serviceEndpoint\":\"https://gateway.ipfs.io/ipfs/bafybeihetj2ng3d74k7t754atv2s5dk76pcqtvxls6dntef3xa6rax25xe\",\"type\":\"LinkedDomains\"},{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#twitter\",\"serviceEndpoint\":\"https://twitter.com/cheqd_io\",\"type\":\"LinkedDomains\"},{\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#linkedin\",\"serviceEndpoint\":\"https://www.linkedin.com/company/cheqd-identity/\",\"type\":\"LinkedDomains\"}],\"verificationMethod\":[{\"controller\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY\",\"id\":\"did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#key1\",\"publicKeyMultibase\":\"zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkYsWCo7fztHtepn\",\"type\":\"Ed25519VerificationKey2020\"}]}",
```
Comment (OR13) [inline packages/did-core-test-server/suites/implementations/dereferencer-cheqd.json:128]: pretty sure `LinkedDomains` and `Ed25519VerificationKey2020` are not in `https://www.w3.org/ns/did/v1`

File: packages/did-core-test-server/suites/implementations/did-cheqd.json [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+        ],
+        "service": [
+          {
+            "id": "did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#website",
+            "serviceEndpoint": "https://www.cheqd.io",
+            "type": "LinkedDomains"
+          },
+          {
+            "id": "did:cheqd:mainnet:zF7rhDBfUt9d1gJPjx7s1JXfUY7oVWkY#non-fungible-image",
+            "serviceEndpoint": "https://gateway.ipfs.io/ipfs/bafybeihetj2ng3d74k7t754atv2s5dk76pcqtvxls6dntef3xa6rax25xe",
```
Comment (OR13) [inline packages/did-core-test-server/suites/implementations/did-cheqd.json:26]: cool example

File: packages/did-core-test-server/suites/implementations/dereferencer-cheqd.json (hunk 1 above)
Comment (Tweeddalex) [reply packages/did-core-test-server/suites/implementations/dereferencer-cheqd.json]: Within https://www.w3.org/ns/did/v1 it explains that the [service](https://www.w3.org/TR/did-core/#dfn-service) type and its associated properties SHOULD be registered in the DID Specification Registries [[DID-SPEC-REGISTRIES](https://www.w3.org/TR/did-core/#bib-did-spec-registries)]

We perhaps should make explicit reference to the DID Spec Registries in the context, where LinkedDomains is one of only two Service Types currently registered: https://www.w3.org/TR/did-spec-registries/#linkeddomains

**Comment by Toktar** [comment]
Hello @OR13,
I added more links for @context. PR is ready for reviewing and merging.
Could you please review it?

**Comment by Tweeddalex** [comment]
Hey @msporny we'd be grateful if you could review this. Thanks 👍

**Review by kdenhartog — APPROVED** [review]

**Review by msporny — APPROVED** [review]

**Pull request merged**